app/editor/src/components/wysiwyg/Wysiwyg.tsx

```diff
@@ -22,8 +22,8 @@ export interface IWysiwygProps<T extends object> {
   required?: boolean;
   /** Whether to show the raw view. */
   viewRaw?: boolean;
-  /** Whether the text area has a height. */
-  hasHeight?: boolean;
+  /** Whether the text area has a custom height. */
+  customHeight?: number;
```

Comment:
Can we name this `height`? and make it a `string`?

Reply:
Updated, thanks